gtclang/test/integration-test/CMakeLists.txt

```diff
@@ -14,9 +14,6 @@
 ##
 ##===------------------------------------------------------------------------------------------===##
 
-set(dummy_source ${CMAKE_CURRENT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/dummy.cpp)
```

Comment:
I thought this was required if `GT` is turned off. Is this no longer the case?

Reply:
This CMakeLists.txt file is only processed if `ENABLE_TESTING=ON` which has already turned on GridTools.